app/code/Magento/Checkout/view/frontend/web/template/minicart/content.html

```diff
@@ -64,24 +68,25 @@
                 <div data-bind="html: getCartParamUnsanitizedHtml('extra_actions')"></div>
             </div>
         </div>
-    </if>
+        <!-- /ko -->
+    <!-- /ko -->
 
-    <if args="getCartParam('summary_count')">
+    <!-- ko if: getCartParam('summary_count') -->
         <strong class="subtitle" translate="'Recently added item(s)'"/>
         <div data-action="scroll" class="minicart-items-wrapper">
             <ol id="mini-cart" class="minicart-items" data-bind="foreach: { data: getCartItems(), as: 'item' }">
-                <each args="$parent.getRegion($parent.getItemRenderer(item.product_type))"
-                      render="{name: getTemplate(), data: item, afterRender: function() {$parents[1].initSidebar()}}"
-                />
+                <!-- ko foreach: $parent.getRegion($parent.getItemRenderer(item.product_type)) -->
+                    <!-- ko template: {name: getTemplate(), data: item, afterRender: function() {$parents[1].initSidebar()}} --><!-- /ko -->
+                <!-- /ko -->
             </ol>
         </div>
-    </if>
+    <!-- /ko -->
 
-    <ifnot args="getCartParam('summary_count')">
+    <!-- ko ifnot: getCartParam('summary_count') -->
         <strong class="subtitle empty"
                 translate="'You have no items in your shopping cart.'"
         />
-        <if args="getCartParam('cart_empty_message')">
+        <!-- ko if: getCartParam('cart_empty_message') -->
             <p class="minicart empty text" text="getCartParam('cart_empty_message')"/>
```

Comment:
This should also be changed to Knockout's binding using `data-bind="text: '...'"`